renderdoc/driver/gl/gl_renderstate.cpp

```diff
@@ -637,6 +637,7 @@ bool GLRenderState::CheckEnableDisableParam(GLenum pname)
       case eGL_PROGRAM_POINT_SIZE:
       case eGL_PRIMITIVE_RESTART:
       case eGL_TEXTURE_CUBE_MAP_SEAMLESS:
+      case eGL_FRAMEBUFFER_SRGB:
```

Comment:
I see what you mean, you were referring to this in the EGL_KHR_gl_colorspace spec:
"GL ES doesn't require GL_FRAMEBUFFER_SRGB enable."

Because we can never create sRGB framebuffers (due to above issue), we have to assume that its always disabled, and therefore I think this new case label is correct.